### PR TITLE
mvp-badge - shows total amount of mvp per user

### PR DESCRIPTION
**Pull request by joseamneto**
## Description
This is a new "feature", basically, it will show the number of MVP awards received by user  example, 3x MVP

## Motivation
A lot of people uses the number of MVPs that they were awarded on LinkedIn, ex 3 Times MVP, therefore I thought that putting this information would help someway, check the image below , note the number "3x and the mvp badge"

## How Has This Been Tested?
Currently, one user can have multiple MVP awards associated with him , this will be added automatically to the filters, ex 2020, 2021, based on this field, the code will count the number of MVP's and show them on the screen
I tested with 3 "Users" and multiple MVP awards  , it would be nice to test with 100 users.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ X ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update (non-breaking change; modified files are limited to the `/docs` directory)

## Checklist:

- [X ] I have read the Contributing guide.
- [ X] My code follows the code style of this project.
- [ X] My code/comments/docs fully adhere to the Code of Conduct.
- [ ] My change is a code change and it requires an update to the documentation.
- [ ] My change is a documentation change and it requires an update to the navigation.

### CODE REVIEW

File: src/Foundation/People/platform/Extenders/CustomSchemaExtender.cs [hunk 1]
```diff
@@ -35,13 +43,23 @@ public CustomSchemaExtender()
...
 			var awards = GetAwardsTags(personItem);
-			if(awards.Any())
+			if (awards.Any())
 			{
 				return awards.First().Key.ToString();
 			}
 			return "";
 		}
 
+		private string GetLatestMvpYearList(Sitecore.Data.Items.Item personItem)
```
Comment (sebastiantecsi) [inline src/Foundation/People/platform/Extenders/CustomSchemaExtender.cs:53]: @joseamneto please rename the function name as it is quite confusing, the name does not reflect what is returning.

File: src/Foundation/People/platform/Extenders/CustomSchemaExtender.cs [hunk 2]
```diff
@@ -35,13 +43,23 @@ public CustomSchemaExtender()
...
+		{
+			var years = string.Empty;
+
+			var awards = GetAwardsTags(personItem);
+			years = awards.Count().ToString();
```
Comment (sebastiantecsi) [inline src/Foundation/People/platform/Extenders/CustomSchemaExtender.cs:58]: @joseamneto , please use another name for the variable year

File: src/Foundation/People/platform/Extenders/CustomSchemaExtender.cs (hunk 1 above)
Comment (joseamneto) [reply src/Foundation/People/platform/Extenders/CustomSchemaExtender.cs]: function renamed to GetAwardsCount

File: src/Foundation/People/platform/Extenders/CustomSchemaExtender.cs (hunk 2 above)
Comment (joseamneto) [reply src/Foundation/People/platform/Extenders/CustomSchemaExtender.cs]: variable "years" removed, returning the awards directly

**Comment by sebastiantecsi** [comment]
Looks good. 

@mrtamasvarga , it will look like this: 

image

And its up to you to have it approve/on the website.

**Comment by robearlam** [comment]
Closed as this repository is being archived.

**Pull request closed without merging**